package.json

```diff
@@ -67,7 +67,6 @@
     "@storybook/addon-actions": "4.1.11",
     "@storybook/addon-links": "4.1.11",
     "@storybook/addon-storyshots": "^4.1.11",
-    "@storybook/addon-viewport": "^4.1.11",
```

Comment:
`viewport` doesn't allow for viewport creation

Reply:
Funny